src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -52,10 +54,23 @@ export default function withAnnotations<P extends object>(
             }
         };
 
+        handlePreviewDestroy = (): void => {
+            this.setState({
+                activeAnnotationId: null,
+                annotation: undefined,
```

Comment:
Should these be nullable fields, instead? It seems weird to explicitly define something as undefined.